packages/angular/projects/clr-angular/src/data/datagrid/datagrid-column.ts

```diff
@@ -322,7 +330,15 @@ export class ClrDatagridColumn<T = any> extends DatagridFilterRegistrar<T, ClrDa
 
   @Output('clrDgSortOrderChange') public sortOrderChange = new EventEmitter<ClrDatagridSortOrder>();
 
+  /**
+   * @deprecated
+   *
+   * Use `sortDirection` to indentify the sort direction
+   */
   public sortIcon: string | null;
+
+  public sortDirection: Directions | null;
```

Comment:
Directions has some values that don't quite make sense if a consumer were to use this. They could still pass 'left' or 'right'.

Perhaps just keep it simple and use only the string literal types we want here + null. e.g

```
public sortDirection: 'up' | 'down' | null;
```

Reply:
Agree!